File: android/BertSpeak/app/src/main/java/chuckcoughlin/bert/service/VoiceConstants.java
/*
 * Copyright (C) 2017 Chuck Coughlin
 *  (MIT License)
 */
package chuckcoughlin.bert.service;

/**
 * This class contains static strings used in conjunction
 * with the Voice Service.
 */
public class VoiceConstants {


    // Broadcast intent categories
    public static final String CATEGORY_FACILITY_STATE = "FacilityState";
    public static final String CATEGORY_SPOKEN_TEXT   = "SpokenText";

    // "extra" data keys
    public static final String KEY_TIERED_FACILITY = "TieredFacility";
    public static final String KEY_FACILITY_STATE = "FacilityState";
    public static final String KEY_SPOKEN_TEXT   = "SpokenText";
}

File: BertSpeak/app/src/main/kotlin/chuckcoughlin/bertspeak/service/VoiceConstants.kt
/*
 * Copyright (C) 2022 Chuck Coughlin
 *  (MIT License)
 */
package chuckcoughlin.bertspeak.service

/**
 * This class contains static strings used in conjunction
 * with the Voice Service.
 */
object VoiceConstants {
    // Broadcast intent categories
    const val CATEGORY_FACILITY_STATE = "FacilityState"
    const val CATEGORY_SPOKEN_TEXT = "SpokenText"

    // "extra" data keys
    const val KEY_TIERED_FACILITY = "TieredFacility"
    const val KEY_FACILITY_STATE = "FacilityState"
    const val KEY_SPOKEN_TEXT = "SpokenText"
}